lblod/frontend-form-builder
# SFB-71: Fix codelijsten

## ID
[ SFB-71](https://binnenland.atlassian.net/browse/SFB-71)

 ## Description

When selecting the input type `dropdown` it didn't show the options for that `concept-scheme`. This was because we do not have all the options in our metaGraph in the local store. Thisdata has to be get in our Virtuoso store. I created a component that will query the database.

As the styling of the `codelijsten` page was not great (I alays copied `code:`)  with it what results in wrong config for the component. Updated this with use of `AuPanel`.


The options will show in the preview but not sure what has to be done when the user has selected that an option. This component is also only in the `form-builder` so if people want to use this component outside of form builder they will NOT be able to for now.

 ## Type of change


 ## How to test

Go to `codelijsten` endpoint select an item and see if the options display on the right. The code configuration for that `concept-scheme` should show in the panel below. Copy the code and past it in the `configuration textArea`  and than select field type to be `dropdown`.

 ## Links to other PR's

 -
